### PR TITLE
Feature/trivy conf

### PR DESCRIPTION
**Pull request by merkata**
This feature adds a trivy config file in the root directory `trivy.yaml` and references it in the integration tests action. That is similar to how it is done on Indico charm.

### CODE REVIEW

**Review by arturo-seijas — DISMISSED** [review]
LGTM

File: trivy.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+exit-code: 2
```
Comment (gregory-schiano) [inline trivy.yaml:1]: What's the goal of returning 2 ?

**Review by gregory-schiano — COMMENTED** [review]
Linting fails due to `SIMULATE_CAN_CONNECT` fix can be found on other projects

**Comment by merkata** [comment]
> Linting fails due to `SIMULATE_CAN_CONNECT` fix can be found on other projects

That's fixed in a different PR, I broke up the chaos testing PR into several PRs and the fix is in the fix/linters branch.

File: trivy.yaml (hunk 1 above)
Comment (merkata) [reply trivy.yaml:1]: Thanks for spotting, I cherry picked the wrong commit :) The `exit-code` needs to be `0`, I was testing earlier to see whether the config overrides what's in the github action for trivy. Thanks!

**Review by arturo-seijas — APPROVED** [review]
LGTM

**Review by gregory-schiano — APPROVED** [review]
LGTM

**Pull request merged**